webpack.config.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-unresolved */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: './src/index.js',
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  devServer: {
+    static: './dist',
+    port: 3000,
+    open: true,
+    hot: true,
```

Comment:
- [ ] please make sure that your application running from my side it doesn't please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/173924789-dbe8f346-5722-4210-8864-0536514e0eb1.png)

Reply:
@youmari Hi again, it runs on my end, I'm using the work dev instead of start, so if you run `npm run dev` it works. The previous reviewer in the webpack project told me everything was okay with that.